Readme.md

```diff
@@ -1,24 +1,55 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128533319/13.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3583)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to apply a filter to a column on the client side
+# GridView for ASP.NET Web Forms - How to apply a filter to a column on the client side
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3583/)**
 <!-- run online end -->
 
+This example demonstrates how to use a text string from the external text box to filter a grid column on the client.
+
+![AutoFilterByColumn](AutoFilterByColumn.gif)
+
+## Overview
+
+Follow the steps below:
+
+1. Create the [Grid View](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control, populate it with columns, and bind it to a data source.
+
+    ```aspx
+    <dx:ASPxGridView ID="grid" ClientInstanceName="grid" runat="server" AutoGenerateColumns="False"
+        DataSourceID="ds" KeyFieldName="CategoryID">
+        <Columns>
+            <dx:GridViewDataTextColumn FieldName="CategoryID" ReadOnly="True" VisibleIndex="0">
+                <EditFormSettings Visible="False" />
+            </dx:GridViewDataTextColumn>
+            <dx:GridViewDataTextColumn FieldName="CategoryName" VisibleIndex="1" />
+            <dx:GridViewDataTextColumn FieldName="Description" VisibleIndex="2" />
+        </Columns>
+    </dx:ASPxGridView>
+    <asp:AccessDataSource ID="ds" runat="server" DataFile="~/App_Data/nwind.mdb"
+        SelectCommand="SELECT [CategoryID], [CategoryName], [Description] FROM [Categories]">
+    </asp:AccessDataSource>
+    ```
+
+2. Add a text box and button. When a user enters a text string in the text box and clicks the button, the grid filters a particular column by the entered text. To enable this behavior, handle the button's client-side [Click](https://docs.devexpress.com/AspNet/js-ASPxClientButton.Click) event. In the handler, call the grid's [AutoFilterByColumn](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.AutoFilterByColumn(column-val)) method and use the text from the text box as a parameter.
```

Comment:
> Add a text box and button.

You urge a reader to add one text box and one button. In this context, "text box" and "button" are two different entities that cannot be joined under the same article.

> When a user enters a text string in the text box and clicks the button, the grid filters a particular column by the entered **text**.

The sentence starts good and well, however, you use "text" as synonym for "text string" in the end. Again, I always stand against overuse of the word "text" as a noun. And this sentence introduces an implied synonym. Instead, you should have used the same noun as in the beginning of your sentence to have an unbroken connection.

> use the **text** from the text box as a parameter.

Likewise, this is not good. "Text" as a noun is a very abstract thing -- it can be literally anything from a sentence to fundamental works of words -- such as books, treatises, religious hymns, or poems. When you speak of parameters, you go down to technical down-to-earth things, therefore, you should use suitable words. In technical sense, you use the current edit value of the text box as a parameter.

Finally, when you speak of parameters, you **pass** a value to your method as a parameter. Yes, you could use "use" here, however, if we can use a more precise word in this context, we should.

Here is my current take on this paragraph:

```suggestion
2. Add a text box and a button. When a user enters a text string in the text box and clicks the button, the grid filters a particular column by the entered string. To enable this behavior, handle the button's client-side [Click](https://docs.devexpress.com/AspNet/js-ASPxClientButton.Click) event. In the handler, call the grid's [AutoFilterByColumn](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.AutoFilterByColumn(column-val)) method and pass the edit value of the text box as a parameter.
```